web/components/tds/tds.postcss

```diff
@@ -91,6 +91,7 @@ body {
   margin: 0;
   min-height: 100vh;
   padding: 0;
+  overflow: hidden;
```

Comment:
not sure why this is necessary, but i was seeing an errant full-page scrollbar after the `FcLayoutDrawerMap` changes